app/sidekiq/delete_old_pii_logs_job.rb

```diff
@@ -3,13 +3,17 @@
 class DeleteOldPiiLogsJob
   include Sidekiq::Job
 
-  sidekiq_options(unique_for: 30.minutes, retry: false)
+  sidekiq_options unique_for: 30.minutes, retry: false
 
   EXPIRATION_TIME = 2.weeks
+  BATCH_SIZE = 10_000
```

Comment:
You mentioned 1,000 records per batch in the PR description. Was that the typo or is 10,000 the typo?

Reply:
Editing the desc now.. decided on 10k to run less queries